src/test/java/uk/gov/justice/probation/courtcasematcher/model/mapper/CaseMapperTest.java

```diff
@@ -109,46 +113,83 @@ void whenMapNewCaseThenCreateNewCaseNoOffences() {
-        assertThat(courtCase.getPnc()).isEqualTo(PNC);
 
-        assertThat(courtCase.getCaseNo()).isEqualTo("123");
-        assertThat(courtCase.getCaseId()).isEqualTo("321321");
-        assertThat(courtCase.getCourtCode()).isEqualTo(COURT_CODE);
-        assertThat(courtCase.getCourtRoom()).isEqualTo("00");
-        assertThat(courtCase.getProbationStatus()).isEqualTo(DEFAULT_PROBATION_STATUS);
-        assertThat(courtCase.getDefendantAddress().getLine1()).isEqualTo("line 1");
-        assertThat(courtCase.getDefendantAddress().getLine2()).isEqualTo("line 2");
-        assertThat(courtCase.getDefendantAddress().getLine3()).isEqualTo("line 3");
-        assertThat(courtCase.getDefendantAddress().getPostcode()).isEqualTo("LD1 1AA");
-        assertThat(courtCase.getDefendantDob()).isEqualTo(DATE_OF_BIRTH);
-        assertThat(courtCase.getDefendantName()).isEqualTo("Mr James BLUNT");
-        assertThat(courtCase.getDefendantSex()).isEqualTo("M");
-        assertThat(courtCase.getSessionStartTime()).isEqualTo(SESSION_START_TIME);
-        assertThat(courtCase.getOffences()).isEmpty();
-        assertThat(courtCase.getGroupedOffenderMatches().getMatches()).hasSize(1);
+        CourtCase courtCaseNew = caseMapper.newFromCourtCaseAndSearchResponse(courtCase, searchResponse);
+
+        assertThat(courtCaseNew).isNotSameAs(courtCase);
+        assertThat(courtCaseNew.getCrn()).isNull();
+        assertThat(courtCaseNew.getGroupedOffenderMatches().getMatches()).hasSize(0);
+    }
+
+    @DisplayName("Map a court case to a new court case when search response has yielded a single match")
+    @Test
+    void givenSingleMatch_whenMapNewFromCaseAndSearchResponse_thenCreateNewCaseWithEmptyListOfMatches() {
+        Match match = Match.builder().offender(Offender.builder()
+            .otherIds(OtherIds.builder().crn(CRN).cro(CRO).pnc(PNC).build())
+            .build())
+            .build();
+
+        CourtCase courtCase = caseMapper.newFromCase(aCase);
+        SearchResponse searchResponse = SearchResponse.builder()
+            .matchedBy(OffenderSearchMatchType.ALL_SUPPLIED)
+            .matches(List.of(match))
+            .probationStatus("Current")
+            .build();
+
+        CourtCase courtCaseNew = caseMapper.newFromCourtCaseAndSearchResponse(courtCase, searchResponse);
+
+        assertThat(courtCaseNew).isNotSameAs(courtCase);
+        assertThat(courtCaseNew.getCrn()).isEqualTo(CRN);
+        assertThat(courtCaseNew.getGroupedOffenderMatches().getMatches()).hasSize(1);
+        OffenderMatch offenderMatch1 = buildOffenderMatch(MatchType.NAME_DOB, CRN, CRO, PNC);
+        assertThat(courtCaseNew.getGroupedOffenderMatches().getMatches()).containsExactly(offenderMatch1);
+    }
+
+    @DisplayName("Map a court case to a new court case when search response has yielded multiple matches")
+    @Test
+    void givenMultipleMatches_whenMapNewFromCaseAndSearchResponse_thenCreateNewCaseWithEmptyListOfMatches() {
```

Comment:
Typo ✏ - `thenCreateNewCaseWithEmptyListOfMatches` list should not be empty

Reply:
Done.